tests/input_file_test.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright (c) 2021 Mira Geoscience Ltd.
+#
+#  This file is part of geoapps.
+#
+#  geoapps is distributed under the terms and conditions of the MIT License
+#  (see LICENSE file at the root of this source code package).
+
+import json
+import os
+
+import pytest
+
+from geoapps.io import InputFile
+
+
+def test_create_work_path():
+    input_file = InputFile("../assets/test.json")
```

Comment:
That's great. Maybe we can store this json inside the test geoh5 file once the test is written.

Reply:
I've changed the way I tested in the latest commits, if I wrote any actual files to disk it uses tmp_path and shouldn't persist on disk after the testing.